python/tvm/auto_scheduler/measure.py

```diff
@@ -697,7 +696,7 @@ def timed_func():
                 args = [ndarray.empty(get_const_tuple(x.shape), x.dtype, ctx) for x in
                         build_res.args]
                 assert tvm.get_global_func("tvm.contrib.random.random_fill", True), \
-                    "Do you enable USE_RANDOM in the config.cmake?"
+                    "Please make sure USE_RANDOM is ON in the config.cmake"
                 random_fill = remote.get_function("tvm.contrib.random.random_fill")
```

Comment:
Here still calls twice.

Reply:
Here `remote.get_function` can not be used  for checking like `tvm.get_global_func`

Reply:
Then we should remove the assert `assert tvm.get_global_func("tvm.contrib.random.random_fill", True)` and use some try catch to capture the error from devices.